### PR TITLE
FOUNDATIONS: Remove File By Id

### LINKED ISSUE #338: FOUNDATIONS: Remove File By Id
# The Ask
We need to develop an OpenAI Service Method that removes a File By Id leveraging OpenAI Broker.

# How to Complete this Task
- Learn what Foundation Services are [here](https://github.com/hassanhabib/The-Standard/blob/master/2.%20Services/2.1%20Foundations/2.1%20Foundations.md#21-foundation-services-broker-neighboring)
- Watch how The Standard Community developed a similar Service for OpenAI [here](https://www.youtube.com/watch?v=-6W_-5cKkfs&list=PLan3SCnsISTTl_MnGP6B78Nfu9Ix8G4mU&index=12) and [here](https://www.youtube.com/watch?v=o8ZdFcScOKU&list=PLan3SCnsISTTl_MnGP6B78Nfu9Ix8G4mU&index=13) and also [here till 1:32:00](https://www.youtube.com/watch?v=OvU6Tate4iI&list=PLan3SCnsISTTl_MnGP6B78Nfu9Ix8G4mU&index=14)
- PR must be developed in a purist approach. One commit for a failing test then another for a passing test as shown in the videos from The Standard Community - PRs that have not met that criteria will be shutdown immediately
- If you have any questions - ask The Standard Community [here](https://discord.com/invite/vdPZ7hS52X)
- You cannot start working on this issue until #337  is completed closed and merged

# How to claim this issue?
- Post on this issue a comment that says: "I claim this issue"
- You will be given 24 hours to complete the issue - if no PR is opened within 24 hours someone else will claim it
- If you have an open PR but is inactive for more than 24 hours from the last review comment we will shutdown your PR

### PR DESCRIPTION
**Pull request by Gummimundur**
Closes #338

### CODE REVIEW

File: Standard.AI.OpenAI.Tests.Unit/Services/Foundations/Files/FileServiceTests.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                Type = objectType,
+                Deleted = GetRandomBoolean()
+            };
+        }
+
+        private static string GetRandomString() =>
+            new MnemonicString().GetValue();
+
+        private static bool GetRandomBoolean() =>
+            Randomizer<bool>.Create();
```
Comment (ElbekDeveloper) [inline Standard.AI.OpenAI.Tests.Unit/Services/Foundations/Files/FileServiceTests.cs:53]: We were using some other method to get random boolean in other tests. We should keep consistency. Please, change this method to be similar to other ones.

Comment (Gummimundur) [reply Standard.AI.OpenAI.Tests.Unit/Services/Foundations/Files/FileServiceTests.cs:53]: @ElbekDeveloper 

```cs
private static bool GetRandomBoolean() =>
    new SequenceGeneratorBoolean().GetValue();
```

Will always return false (see #247).  `AIModelServiceTests` is still using `SequenceGeneratorBoolean` but that should be updated

Comment (hassanhabib) [reply Standard.AI.OpenAI.Tests.Unit/Services/Foundations/Files/FileServiceTests.cs:53]: @Gummimundur create an issue for it. @ElbekDeveloper take care of resolving it please.

Comment (Gummimundur) [reply Standard.AI.OpenAI.Tests.Unit/Services/Foundations/Files/FileServiceTests.cs:53]: @hassanhabib @ElbekDeveloper Issue created. See #347

**Pull request merged**